runtimes/node-18.0/server.js

```diff
@@ -30,13 +30,35 @@ const server = micro(async (req, res) => {
         headers[header.toLowerCase()] = req.headers[header];
     });
 
+    const scheme = (req.headers['x-forwarded-proto'] ?? 'http');
+    const host = req.headers['host'].includes(':') ? req.headers['host'].split(':')[0] : req.headers['host'];
+    const port = +(req.headers['host'].includes(':') ? req.headers['host'].split(':')[1] : '80');
```

Comment:
Should we always assume `http` and port `80` if none are found?

Reply:
There has to be a protocol, and there has to be a port. those sound as reasonable defaults.

> If the port is empty or not given, port 80 is assumed (HTTP RFC)

Reply:
I think we should use 443 as the default if the scheme is `https`

Reply:
Fixed, updated tests.